src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -1,12 +1,12 @@
 // Copyright (C) 2018-2024 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 
-#include "node/include/model_wrap.hpp"
-
 #include "node/include/addon.hpp"
 #include "node/include/errors.hpp"
 #include "node/include/helper.hpp"
+#include "node/include/model_wrap.hpp"
```

Comment:
Please revert this change and move `#include "node/include/model_wrap.hpp"` to the top.
Probably you are using a different formatting tool than is specified [here](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/coding_style.md)